deepinv/loss/metric/perceptual.py

```diff
@@ -0,0 +1,77 @@
+from deepinv.loss.metric.metric import import_pyiqa, Metric
+
+
+class LPIPS(Metric):
+    r"""
+    Learned Perceptual Image Patch Similarity (LPIPS) metric.
+
+    Computes the perceptual similarity between two images, based on a pre-trained deep neural network.
+    Uses implementation from `pyiqa <https://pypi.org/project/pyiqa/>`_.
+
+    See docs for ``forward()`` below for more details.
+
+    :Example:
+
+    >>> from deepinv.utils.demo import get_image_url, load_url_image
+    >>> from deepinv.loss.metric import LPIPS
+    >>> ();m = LPIPS();() # doctest: +ELLIPSIS
```

Comment:
what is this line doing?

Reply:
It's because pyiqa is annoyingly written (they use conditional print statements in their code instead of warnings) + doctest is a bit fiddly so this is the only hacky way to capture the print output